src/Settings.php

```diff
@@ -36,10 +42,186 @@ public function __construct( $options_prefix ) {
 
 			$this->set_options_prefix( $options_prefix );
 
+			add_action( 'tec_events_settings_tab_display', [ $this, 'add_settings_tab' ] );
+
 			// Add settings specific to OSM
 			add_action( 'admin_init', [ $this, 'add_settings' ] );
 		}
 
+		/**
+		 * Add a tab to the Display settings.
+		 *
+		 * @since 1.2.0
+		 *
+		 * @param \Tribe__Settings_Tab $display_tab Object containing the data for the settings tab.
+		 *
+		 * @return void
+		 */
+		public function add_settings_tab( $display_tab ) {
+			$labels_tab = new \Tribe__Settings_Tab(
+				'display-labels-tab',
+				esc_html_x( 'Labels', 'Settings tab label', 'tribe-ext-relabeler' ),
+				[
+					'priority' => 20,
+					'fields'   => $this->add_settings(),
+				]
+			);
+
+			$display_tab->add_child( $labels_tab );
+		}
+
+		/**
+		 * Add the settings to the settings tab.
+		 *
+		 * @since 1.2.0
+		 *
+		 * @return array
+		 */
+		public function add_settings() {
+			$settings = [
+				'tec-settings-form__header-block' => ( new Div( new Classes( [ 'tec-settings-form__header-block', 'tec-settings-form__header-block--horizontal' ] ) ) )->add_children(
+					[
+						new Heading(
+							_x( 'Labels', 'Labels settings header', 'tribe-ext-relabeler' ),
+							2,
+							new Classes( [ 'tec-settings-form__section-header' ] )
+						),
+						( new Paragraph( new Classes( [ 'tec-settings-form__section-description' ] ) ) )->add_children(
+							[
+								new Plain_Text(
+									__(
+										"The following fields allow you to change the default labels. Inputting something other than the default will change that word everywhere it appears. ",
+										'tribe-ext-relabeler'
+									)
+								),
+								new Plain_Text(
+									// Translators: Beginning of the sentence "Remember to resave the permalinks after doing any changes.
+									__( 'Remember to ', 'tribe-ext-relabeler' ),
+								),
+								new Link(
+										admin_url( 'options-permalink.php' ),
+										// Translators: Middle of the sentence "Remember to resave the permalinks after doing any changes.
+										__( 'resave the permalinks', 'tribe-ext-relabeler' ),
+								),
+								new Plain_Text(
+								// Translators: End of the sentence "Remember to resave the permalinks after doing any changes.
+									__( ' after doing any changes.', 'tribe-ext-relabeler' ),
+								),
+							]
+						),
+					]
+				),
+			];
+
+			$fields_setup = [
+				'labels_heading' => [
+					'type' => 'html',
+					'html' => '<h3 class="tec-settings-form__section-header tec-settings-form__section-header--sub">' . esc_html__( 'Labels', 'tribe-ext-relabeler' ) . '</h3>',
+				],
+				'label_event_single' => [
+					'type'            => 'text',
+					'label'           => esc_html__( 'Event', 'the-events-calendar' ),
```

Comment:
Noticed we are using different language domain here. I was expecting `tribe-ext-relabeler`

Reply:
@dpanta94 using `the-events-calendar` here, so non-English users don't need to bother translating it separately. If it's translated in TEC, it'll be translated in the extension as well.